pom.xml

```diff
@@ -8,7 +8,7 @@
 	<parent>
 		<groupId>org.springframework.cloud.task.app</groupId>
 		<artifactId>task-app-starters-build</artifactId>
-		<version>2.0.1.BUILD-SNAPSHOT</version>
+		<version>2.0.0.BUILD-SNAPSHOT</version>
```

Comment:
Is there any reason why we downgrade the version from `2.0.1` to `2.0.0`

Reply:
Great question!   The last release put the wrong version number for the task-app-starters-build.